.changeset/tough-tomatoes-protect.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fuel-ts/account": minor
+"@fuel-ts/errors": minor
+---
+
+- Handling `SqueezedOut` status update when calling `submitAndAwait` subscription at `Provider` `sendTransaction`
+- Handling `SqueezedOut` status update when calling `submitAndAwait` subscription at `Provider.sendTransaction`
+- Handling `SqueezedOut` status update when calling `statusChange` subscrition at `TransactionResponse.fetch`
```

Comment:
Hmm, seems like something went wrong with the suggestion. Here's the correct changeset suggestion:

- Handling `SqueezedOut` status update when calling `submitAndAwait` subscription at `Provider.sendTransaction`
- Handling `SqueezedOut` status update when calling `statusChange` subscrition at `TransactionResponse.waitForResult`